app/assets/javascripts/items.js.coffee

```diff
@@ -4,3 +4,15 @@ jQuery ->
     regexp = new RegExp($(this).data('id'), 'g')
     $(this).before($(this).data('fields').replace(regexp, time))
     event.preventDefault()
+
+jQuery ->
+  $('#items').dataTable
+    columnDefs: [
+                  ## https://datatables.net/extensions/responsive/priority
+                  { responsivePriority: 1, targets: [0, 1, 2, 3, 8, 9 ] },
```

Comment:
This is a good start, but we'll need to change the view itself. In a different PR, we should remove the `serial X` columns, as they don't really apply anymore. Also, I'd like to remove the `Available` status column, and just colorize the row itself.
https://datatables.net//forums/discussion/34882